### PR TITLE
LPS-139879 modifying the mailing by headless-form

### PR DESCRIPTION
**Pull request by antonio91dev**

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/notification/DDMFormEmailNotificationSender.java [hunk 1]
```diff
@@ -476,14 +476,14 @@ protected String getUserName(
 	}
 
 	protected String getViewFormEntriesURL(
-			DDMFormInstance ddmFormInstance, ThemeDisplay themeDisplay)
+			DDMFormInstance ddmFormInstance, ServiceContext serviceContext)
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/notification/DDMFormEmailNotificationSender.java:479]: @antonio91dev please order the parameters to be consistent with the other methods `getEmailBody`, `createTemplate`, `createMailMessage` and `populateParameters `.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/notification/DDMFormEmailNotificationSender.java [hunk 2]
```diff
@@ -500,14 +500,14 @@ protected String getViewFormEntriesURL(
 	protected String getViewFormURL(
 			DDMFormInstance ddmFormInstance,
 			DDMFormInstanceRecord ddmFormInstanceRecord,
-			ThemeDisplay themeDisplay)
+			ServiceContext serviceContext)
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/notification/DDMFormEmailNotificationSender.java:503]: @antonio91dev please order the parameters to be consistent with the other methods `getEmailBody`, `createTemplate`, `createMailMessage` and `populateParameters `.

**Comment by carolmariaabb** [comment]
ci:test:sf

**Review by carolmariaabb — CHANGES_REQUESTED** [review]
@antonio91dev please see my comments below. In addition, can you please fix the source formatter issues? 
Thanks

**Comment by antonio91dev** [comment]
@carolmariaabb  a new commit was added applying the changes

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
> ### ❌ ci:test:relevant - 0 out of 1 jobs passed in 14 minutes
> Click here for more details.
> #### Base Branch:
> Branch Name: [master](https://github.com/liferay/liferay-portal/tree/master)Branch GIT ID: [596776d659b936539c7dfed23e61b35bc51a8bb9](https://github.com/liferay/liferay-portal/commit/596776d659b936539c7dfed23e61b35bc51a8bb9)
> 
> #### Upstream Comparison:
> Branch GIT ID: [596776d659b936539c7dfed23e61b35bc51a8bb9](https://github.com/liferay/liferay-portal/commit/596776d659b936539c7dfed23e61b35bc51a8bb9)Jenkins Build URL: [Acceptance Upstream DXP (master) #2364](https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//2364)
> 
> **ci:test:relevant - 0 out of 1 jobs PASSED**
> ##### For more details click [here](https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/411/jenkins-report.html).
> #### Failures unique to this pull:
> 1. ##### [test-portal-acceptance-pullrequest(master)](https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/411/)###### Job Results:0 Jobs Passed.1 Job Failed.```
>         [exec] > Task :apps:dynamic-data-mapping:dynamic-data-mapping-service:compileJava FAILED
>         [exec] 
>         [exec] > Task :apps:commerce:commerce-tax-engine-fixed-api:autoUpdateXml SKIPPED
>         [exec] 
>         [exec] > Task :apps:commerce:commerce-cart-taglib:packageRunBuild
>         [exec] Using pre-existing NODE_ENV: "test"
>         [exec] Successfully compiled 0 files with Babel.
>         [exec] Bundling took 130ms
>         [exec] BUILD SASS: No scss files found.
>         [exec] Minification summary:
>         [exec]   Files processed:   0
>         [exec]   Files with errors: 0
>         [exec]   Before size:       0
>         [exec]   After size:        0
>         [exec]   Delta:             0
>         [exec]   Elapsed:           0.00
>         [exec] Received 0
>         [exec] Done in 2.83s.
>         [exec] 
>         [exec] > Task :apps:commerce:commerce-service:compileJava
>         [exec] Note: Some input files use or override a deprecated API.
>         [exec] Note: Recompile with -Xlint:deprecation for details.
>         [exec] Note: Some input files use unchecked or unsafe operations.
>         [exec] Note: Recompile with -Xlint:unchecked for details.
>         [exec] 
>         [exec] > Task :apps:change-tracking:change-tracking-web:packageRunBuild
>         [exec] Bundling took 26s
>         [exec] BUILD SASS: Built build/node/packageRunBuild/resources/.sass-cache/publications/css/ChangeTrackingIndicator.css
>         [exec] BUILD SASS: Built build/node/packageRunBuild/resources/.sass-cache/publications/css/main.css
>         [exec] Minification summary:
>         [exec]   Files processed:   2439
>         [exec]   Files with errors: 1234
>         [exec]   Before size:       7698901
>         [exec]   After size:        4757491
>         [exec]   Delta:             2941410
>         [exec]   Elapsed:           9.02
>         [exec] Received 0
>         [exec] Done in 41.08s.
>         [exec] Gradle build finished at 2021-09-29 13:33:15.074.
>         [exec] 1332 actionable tasks: 1270 executed, 62 up-to-date
>         [exec] 
>         [exec] 
>         [exec] FAILURE: Build failed with an exception.
>         [exec] 
>         [exec] * What went wrong:
>         [exec] Execution failed for task ':apps:dynamic-data-mapping:dynamic-data-mapping-service:compileJava'.
>         [exec] > Compilation failed; see the compiler error output for details.
>         [exec] 
>         [exec] * Try:
>         [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
>         [exec] 
>         [exec] * Exception is:
>    ```
> 
> #### For upstream results, click [here](https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)).

@antonio91dev Can you please check the CI failures? It seems to be related to your changes.

```
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:dynamic-data-mapping:dynamic-data-mapping-service:compileJava'.
     [exec] > Compilation failed; see the compiler error output for details.
```

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @antonio91dev, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:reevaluate:1137401_6360

**Comment by carolmariaabb** [comment]
ci:reevaluate:1137401_6360

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by antonio91dev** [comment]
Hi
@liferay-forms  no , these are not errors related to my changes

**Comment by liferay-forms** [comment]
Hey @antonio91dev, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Review by carolmariaabb — APPROVED** [review]

**Comment by liferay-forms** [comment]
Hey @antonio91dev, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:reevaluate:1297401_387

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @antonio91dev, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**